src/darwin/Framework/CHIP/CHIPPersistentStorageDelegateBridge.h

```diff
@@ -35,9 +35,8 @@ class CHIPPersistentStorageDelegateBridge : public chip::PersistentStorageDelega
     CHIP_ERROR SyncDeleteKeyValue(const char * key) override;
 
 private:
-    id<CHIPPersistentStorageDelegate> mDelegate;
-
-    dispatch_queue_t mWorkQueue;
+    _Nullable id<CHIPPersistentStorageDelegate> mDelegate;
+    _Nullable dispatch_queue_t mWorkQueue;
```

Comment:
This one does not make sense to me: those are set non-null in the constructor, generally.  What was the error here?

Reply:
```
2022-04-25 23:06:56 INFO    TIDY ../../src/darwin/Framework/CHIP/CHIPPersistentStorageDelegateBridge.mm: /Users/andrei/devel/connectedhomeip/out/default/../../src/darwin/Framework/CHIP/CHIPPersistentStorageDelegateBridge.mm:56:1: error: nil assigned to a pointer which is expected to have non-null value [clang-analyzer-nullability.NullPassedToNonnull,-warnings-as-errors]
{
^
/Users/andrei/devel/connectedhomeip/out/default/../../src/darwin/Framework/CHIP/CHIPPersistentStorageDelegateBridge.mm:56:1: note: nil assigned to a pointer which is expected to have non-null value
```

Agree that not a good error, yet this is what I have. Guessing mWorkQueue is first started up as nil when the class is constructed. Not sure. I am not trying to fix the actual errors for now, just making the linter happy. I believe actual fixes if these are real bugs (some may be) should be as a followup.